adding_species_manuscript.tex

```diff
@@ -167,15 +167,11 @@ \section*{Abstract}
 The initial version of \stdpopsim focused on establishing this framework using 
 six well-characterized model species.
 Here, we report on major updates made in the new release of \stdpopsim (version 0.2),
-which includes a significant expansion of the species catalog and substantial additions to simulation capabilities that make it possible to simulate a broader range of species.
+which includes a significant expansion of the species catalog and substantial additions to simulation capabilities.
 Features added to improve the realism of the simulated genomes include non-crossover recombination and provision of species-specific genomic annotations.
 Through community-driven efforts, we expanded the number of species in the catalog by more than three-fold and broadened coverage across the tree of life.
-Our experience throughout this process was that people are keen to put in the time and effort to include their study species,
-but that simple, clear guidance is vital.
-Consequently, our intention with this paper is also to provide a learning
-modality to meet that need,
```

Comment:
A "learning modality" is a *way* of learning things, whereas we have a set of guidelines. Also, the first sentence here seemed a bit condescending, so I rephrased.

Reply:
I agree with changing "modality" to "guidelines", but I like preceding this with a statement that ties these guidelines to a need that came up during the expansion of the catalog. This gets a bit lost in the shorter version. I couldn't think of a good compromise. @lauterbur? Do you have a suggestion here?

Reply:
See suggestion below.